src/ai/backend/cli/interaction.py

```diff
@@ -0,0 +1,116 @@
+import ipaddress
+import os
+
+
+def ask_host(prompt: str, default: str = "127.0.0.1") -> str:
+    try:
+        default = default.replace("localhost", "127.0.0.1")
+        _ = ipaddress.ip_address(default)
+    except ValueError:
+        raise ValueError("IP host must be given as 127.0.0.1")
```

Comment:
I don't understand the intention of this error check.
Isn't it checking the IP address format later again?
Also, what if the user wants to put DNS hostnames?
I think it would be better to have an option flag to accept only IP addresses or not, to allow/deny human-readable hostnames.